src/AppBundle/Controller/Report/ReportController.php

```diff
@@ -78,16 +78,9 @@ public function indexAction(Request $request)
         $coDeputies = !empty($client) ? $client->getCoDeputies() : [];
 
         $reports = $client ? $client->getReports() : [];
-        arsort($reports);
-
-        $reportActive = null;
-        $reportsSubmitted = [];
-        foreach ($reports as $currentReport) {
-            if ($currentReport->getSubmitted()) {
-                $reportsSubmitted[] = $currentReport;
-            } else {
-                $reportActive = $currentReport;
-            }
+        $reportsSubmitted = $client ? $client->getSubmittedReports() : [];
```

Comment:
@jackgoodby the method doesn't do the arsort, so you are altering the logic here. You might need to re-add it on $reportsSubmitted (not inside the method, as it's not always wanted I think). Might be not neeed, but you need to check the usage to be sure

Reply:
arsort now line 82